walletunlocker/service.go

```diff
@@ -277,15 +264,6 @@ func (u *UnlockerService) UnlockWallet(ctx context.Context,
 		return nil, err
 	}
 
-	// Attempt to create a password for the macaroon service.
```

Comment:
Nice, this'll speed up the start time as well, also there's another pending PR to remove the extra invocation of scrypt when we're unlocking the wallet.